python/katana/cpp/libstd/optional.pxd

```diff
@@ -0,0 +1,34 @@
+from libcpp cimport bool
```

Comment:
You copied this from a PR right? If so, you **need** to cite the source and make the license clear. (which is presumably the same license as Cython as a whole.

Reply:
I did. It was not explicitly licensed and since it was never merged into the main repo, I'm not sure if it falls under the license?